stores/auctionsStore.js

```diff
@@ -263,6 +266,22 @@ const [useAuctionsStore, updateState] = create((set, get) => ({
     }, 500);
   },
 
+  fetchAllFlip: async (maker, ilk) => {
+    const service = maker.service(AUCTION_DATA_FETCHER);
+
+    // AuctionIDs will clash unless we filter by ilk
+    const auctions = (await service.fetchFlipAuctions()).filter(
+      x => x.ilk === ilk
+    );
+    const transformedAuctions = await transformEvents(
+      auctions,
+      service,
+      'flip',
+      ilk
+    );
+    set({ auctions: transformedAuctions });
```

Comment:
Can we store flip auctions under a different key maybe? User should be able to navigate between pages and this store will hold mixed data...

Reply:
Yeah, that makes sense. One issue with flip auctions is that each ilk will be numbered sequentially too, so there will be ID clashes which causes incorrect merging of ETH & BAT events with the same auctionId key. Would it help to have separate keys for each ilk, or should we handle that with a filter?